src/cli/task.rs

```diff
@@ -202,16 +208,23 @@ pub fn execute(args: Args) -> miette::Result<()> {
                 task,
             );
         }
-        Operation::List => {
+        Operation::List(args) => {
             let tasks = project.task_names(Some(Platform::current()));
             if tasks.is_empty() {
                 eprintln!("No tasks found",);
```

Comment:
What happens if there are no tasks and you try to complete? Will you complete "No"?

Reply:
The difference is in the `eprintln` vs `println` the autocompletion will write the `stderr` to `/dev/null` so it won't complete anything.

Reply:
Makes sense!